src/pages/index.js

```diff
@@ -1,15 +1,95 @@
-import Head from 'next/head';
+import {useState} from 'react';
+import {v4 as uuid} from 'uuid';
 
-import Layout from '../components/Layout';
+import Counter from '../components/Counter';
+import TodoList from '../components/toList.js';
+import useStore from '../components/useStore';
 
-export default function HomePage() {
+export default function Home() {
+	const todos = useStore(state => state.todos);
+	const setTodos = useStore(state => state.setTodos);
+
+	const [searchQuery, setSearchQuery] = useState('');
+
+	const handleDeleteTodo = id => {
+		setTodos(todos.filter(todo => todo.id !== id));
+	};
+
+	const handleAddtodo = event => {
+		if (event.key === 'Enter') {
+			setTodos([...todos, {id: uuid(), title: event.target.value, done: false}]);
+			event.target.value = '';
+		}
+	};
+
+	// Abhacken von Todos
+	const toggleTodo = id => {
+		setTodos(
+			todos.map(todo => {
+				if (todo.id === id) {
+					return {...todo, done: !todo.done};
+				}
+				return todo;
+			})
+		);
+	};
+
+	const activeTodos = todos.filter(({done}) => !done);
+	const doneTodos = todos.filter(({done}) => done);
+	const filterTodos = todos.filter(todo =>
+		todo.title.toLowerCase().includes(searchQuery.toLowerCase())
+	);
+
+	// Suche von Todos
 	return (
-		<Layout>
-			<Head>
-				<title key="title">My Project</title>
-				<meta key="description" name="description" content="This is my project" />
-			</Head>
-			<h1>Home</h1>
-		</Layout>
+		<div className="main">
```

Comment:
If this is a `main` element you should use the `main`tag. We are also working with styled components and not with css files. You should remove your classes.